src/frontends/ir/tests/rt_info_deserialization.cpp

```diff
@@ -4,36 +4,32 @@
 
 #include <gtest/gtest.h>
 
-#include <sstream>
-#include <string>
-#include <memory>
-
 #include <inference_engine.hpp>
 #include <memory>
 #include <ngraph/opsets/opset8.hpp>
+#include <sstream>
 #include <string>
 #include <transformations/rt_info/fused_names_attribute.hpp>
-#include <transformations/rt_info/old_api_map_order_attribute.hpp>
 #include <transformations/rt_info/old_api_map_element_type_attribute.hpp>
-#include "openvino/frontend/manager.hpp"
+#include <transformations/rt_info/old_api_map_order_attribute.hpp>
+
 #include "common_test_utils/graph_comparator.hpp"
 #include "ie_blob.h"
 #include "ie_precision.hpp"
 #include "ngraph/node.hpp"
 #include "ngraph/op/parameter.hpp"
+#include "ngraph/pass/manager.hpp"
 #include "ngraph/pass/pass.hpp"
 #include "ngraph/shape.hpp"
 #include "ngraph/type/element_type.hpp"
 #include "ngraph/variant.hpp"
-#include "ngraph/pass/manager.hpp"
-#include "openvino/runtime/core.hpp"
```

Comment:
Please remove old legacy and redundant headers

Reply:
done